packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -76,6 +84,7 @@
           "type": "string"
         },
         "oauth_client_id": {
+          "description": "The unique public identifier for the application.",
```

Comment:
This is not quite right. This is the OAuth client ID for a RP integrated with FxA.

Reply:
Aah, I was having trouble with this one - thanks!